src/main/target/REVO/target.h

```diff
@@ -67,14 +71,33 @@
 #define BEEPER                  NONE
 #endif
 
-// PC0 used as inverter select GPIO
-#define INVERTER_PIN_UART1      PC0
+// UART inverters
+#ifdef AIRBOTF4SD
+#  define INVERTER_PIN_UART6      PD2
```

Comment:
@mikeller : Good point. My calculation was wrong, if is 4% (1/22) total, 0.7% without generated files (probably mostly my commits)

And as for default ... mixing both styles may be best. With complicated `#if`s indentation helps with code readability. But with simple, single level cases not indenting is easier to read ...